### PR TITLE
docs: add create index docs

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# CREATE INDEX
+
+## Syntax
+
+```sql
+CreateIndextmt ::=
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md]: CreateIndexStmt

File: docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    | OptionsListPrefix "," OptionEntry
+
+OptionEntry ::=
+    Identifier "=" Identifier
+
+```
+
+**Description**
+
+The `CREATE INDEX` statement is used to create a new index on OpenMLDB. If there are data in the table, some asynchronous jobs will be executed to load data. We can see the job status in `ns_client` with [`showopstatus`](../../../maintain/cli.md) command.
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md]: some asynchronous jobs will be executed to load data
=>
data will be loaded asynchronously

File: docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+CREATE INDEX index2 ON t5 (col2);
+-- SUCCEED
+```
+
+We can also set ts column as below:
+```SQL
+CREATE INDEX index3 ON t5 (col3) OPTIONS (ts=ts1, ttl_type=absolute, ttl=30d);
+-- SUCCEED
+```
+More details about `TTL` and `TTL_TYPE` refer [here](./CREATE_TABLE_STATEMENT.md)
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/CREATE_INDEX_STATEMENT.md]: Please refer [here](./CREATE_TABLE_STATEMENT.md) for more details about `TTL` and `TTL_TYPE`.

File: docs/en/reference/sql/ddl/DROP_INDEX_STATEMENT.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# CREATE INDEX
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/DROP_INDEX_STATEMENT.md]: DROP INDEX

File: docs/en/reference/sql/ddl/DROP_INDEX_STATEMENT.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## Syntax
+
+```sql
+CreateIndextmt ::=
```
Comment (lumianph) [inline docs/en/reference/sql/ddl/DROP_INDEX_STATEMENT.md]: DropIndexStmt

File: docs/zh/reference/sql/ddl/CREATE_INDEX_STATEMENT.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+# CREATE INDEX
+
+## 语法
+
+```sql
+CreateIndextmt ::=
```
Comment (lumianph) [inline docs/zh/reference/sql/ddl/CREATE_INDEX_STATEMENT.md]: CreateIndexStmt

File: docs/zh/reference/sql/ddl/DROP_INDEX_STATEMENT.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# CREATE INDEX
```
Comment (lumianph) [inline docs/zh/reference/sql/ddl/DROP_INDEX_STATEMENT.md]: DROP INDEX

File: docs/zh/reference/sql/ddl/DROP_INDEX_STATEMENT.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+## 语法
+
+```sql
+CreateIndextmt ::=
```
Comment (lumianph) [inline docs/zh/reference/sql/ddl/DROP_INDEX_STATEMENT.md]: DropIndexStmt

**Review by lumianph — CHANGES_REQUESTED** [review]

**Comment by lumianph** [comment]
@dl239 you also need to add the entry in the index.rst 
https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/reference/sql/ddl/index.rst
https://github.com/4paradigm/OpenMLDB/blob/main/docs/zh/reference/sql/ddl/index.rst

**Review by lumianph — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**